src/cmd-remote-build-container

```diff
@@ -101,9 +102,9 @@ def main():
     # Set the REGISTRY_AUTH_FILE env var if user passed --authfile
     if args.authfile:
         os.environ["REGISTRY_AUTH_FILE"] = args.authfile
-    # Check for requisite env vars
-    if environ.get('CONTAINER_HOST') is None or environ.get('CONTAINER_SSHKEY') is None:
-        sys.exit('You must have CONTAINER_HOST and CONTAINER_SSHKEY environment variables setup')
+#   # Check for requisite env vars
+#   if environ.get('CONTAINER_HOST') is None or environ.get('CONTAINER_SSHKEY') is None:
+#       sys.exit('You must have CONTAINER_HOST and CONTAINER_SSHKEY environment variables setup')
```

Comment:
Do we want it here commented?

Reply:
this was a mistake. Thanks for pointing it out.